staging/src/k8s.io/client-go/tools/cache/reflector.go

```diff
@@ -172,6 +172,7 @@ func (r *Reflector) ListAndWatch(stopCh <-chan struct{}) error {
 		var err error
 		listCh := make(chan struct{}, 1)
 		panicCh := make(chan interface{}, 1)
+		defer close(panicCh)
```

Comment:
it's okay to leave a channel open w/o closing it, it'll be GC'd automatically. https://groups.google.com/forum/#!msg/golang-nuts/pZwdYRGxCIk/qpbHxRRPJdUJ

Reply:
Looking at other channels in the same class, the practice is to close them through defer.

It would good to follow the same practice.

Reply:
interesting point. good luck :)

Reply:
I don't see the point of closing this channel. The channels that are closed in this file are ones that need to be closed to communicate to receivers that no future messages will be sent on the channel.